Syllabus.md

```diff
@@ -0,0 +1,272 @@
+Stack Club Course Syllabus
+==========================
+
```

Comment:
A hyperlinked table of contents would be useful here I think - we can make one using html anchors, like [this FAQ](https://github.com/drphilmarshall/GettingStarted/blob/master/README.md).